hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/FrameRunner.java

```diff
@@ -125,6 +125,8 @@ private long effectiveGasUsed(final long gasLimit, @NonNull final MessageFrame f
                 * Math.min(frame.getSelfDestructs().size(), nominalUsed / gasCalculator.getMaxRefundQuotient());
         nominalUsed -= (selfDestructRefund + frame.getGasRefund());
         final var maxRefundPercent = contractsConfigOf(frame).maxRefundPercentOfGasLimit();
+        // include the base gas consumption of 21_000 in the final calculation
+        nominalUsed = nominalUsed < 21_000L && nominalUsed > 0 ? nominalUsed + 21_000L : nominalUsed;
```

Comment:
Please use a constant instant of a magic number

Reply:
This piece of code will be completely removed regarding the @tinker-michaelj comment above.